partner_bank_active/views/res_partner_bank.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017-2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <!--FORM view-->
+    <record model="ir.ui.view" id="res_partner_bank_form_view">
+        <field name="name">res.partner.bank.form (in partner_bank_active)</field>
+        <field name="model">res.partner.bank</field>
+        <field name="inherit_id" ref="base.view_partner_bank_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[group[field[@name='sequence']]]" position="before">
```

Comment:
So this can become:
```<field name="sequence" position="before">```
and the also ofcourse replace end `</xpath>' with '</field>`.

Reply:
@NL66278 Nope. The xpath expression used, will place the new field before the <group> element and not the 'sequence' field. The expression mean 'group element that has a group element that has a field element that has name property='sequence'

Reply:
@rousseldenis I see, I misread it...